features/AssetController.feature

```diff
@@ -17,6 +17,33 @@ Feature: Device Manager asset controller
       | _id      | "outils-PERFO-asset_01" |
     Then The document "engine-kuzzle":"assets":"outils-PERFO-asset_01" does not exists
 
+  Scenario: Delete a linked asset
+    When I successfully execute the action "device-manager/asset":"create" with args:
+      | engineId       | "engine-ayse" |
+      | body.type      | "outils"        |
+      | body.model     | "PERFO"         |
+      | body.reference | "asset_02"      |
+    Then The document "engine-ayse":"assets":"outils-PERFO-asset_02" exists
+    When I successfully execute the action "device-manager/device":"linkAsset" with args:
+        | _id                            | "DummyTemp-attached_ayse_unlinked" |
+        | assetId                        | "outils-PERFO-asset_02"             |
+        | body.metadata.index  | "engine-ayse"         |
+    Then The document "engine-ayse":"devices":"DummyTemp-attached_ayse_unlinked" content match:
+      | assetId | "outils-PERFO-asset_02" |
+    And  The document "device-manager":"devices":"DummyTemp-attached_ayse_unlinked" content match:
+      | assetId | "outils-PERFO-asset_02" |
```

Comment:
Same for those steps because this scenario is not testing the asset linking. It should assume that this part is working since another scenario is ensuring this

Reply:
done